prog/vm/github_runner.rb

```diff
@@ -135,6 +135,23 @@ def before_run
   label def wait_vm
     nap 5 unless vm.strand.label == "wait"
     register_deadline(:wait, 10 * 60)
+    hop_create_runner_user
+  end
+
+  label def create_runner_user
```

Comment:
Do you have an estimate of how much it increases our provisioning time?

Reply:
As I tested running that script it's runtime is on milliseconds level. Though, the biggest portion will probably opening a new ssh to a vm. 

So, it would be at most what we see for setup_environment. Setup_environment's 99th percentile is ~10 seconds and I expect this to take few seconds less at the worst case. I'm planning to follow how it affects the provisioning time and might combine that script with the script on setup_environment accordingly (which I don't want as it means combining two different things on the same state).